src/endpoints/Artist.ts

```diff
@@ -1,20 +1,58 @@
 import { ReadWriteBaseClient } from '../client/ReadWriteBaseClient';
-import { Artist as ArtistProfile, ArtistsAlbumPageResult } from '../types';
+import { 
+  Artist as ArtistProfile,
+  ArtistsAlbumPageResult,
+  Artists as ArtistsProfile,
+  Tracks as TracksDetail,
+} from '../types';
+import { joinIdsArrayToString } from '../utils';
 import { SpotifiedPaginator } from '../paginators';
 import { SimplifiedAlbum } from '../types/album.types';
 
 export class Artist extends ReadWriteBaseClient {
+    /**
+   * Get Spotify catalog information for a single artist identified by their unique Spotify ID
+   * https://developer.spotify.com/documentation/web-api/reference/get-an-artist
+   */
   getArtist(id: string) {
     return this.get<ArtistProfile>(`/artists/${id}`);
   }
 
+  /**
+   * Get Spotify catalog information for several artists based on their Spotify IDs
+   * https://developer.spotify.com/documentation/web-api/reference/get-multiple-artists
+   */
+  getArtists(ids: string[]) {
+    return this.get<ArtistsProfile>(`/artists?ids=${joinIdsArrayToString(ids)}`);
+  }
+
+  /**
+   * Get Spotify catalog information about an artist's albums
+   * https://developer.spotify.com/documentation/web-api/reference/get-an-artists-albums
+   */
   getArtistAlbums(artistId: string) {
```

Comment:
Remove this endpoint and write it from scratch just like you have done for other endpoints